core/src/main/scala/io/gearpump/cluster/ClusterConfig.scala

```diff
@@ -142,7 +142,7 @@ object ClusterConfig {
 
     if (!akka.util.Helpers.isWindows) {
 
-      // Change the akka.scheduler.tick-duration to 10ms for Linux or Mac
+      // Change the akka.scheduler.tick-duration to 1 ms for Linux or Mac
```

Comment:
That's different situations.
When the cluster config is loaded, it will check the environment and ```akka.scheduler.tick-duration``` to make sure the ActorSystem can be launched correctly.

However, when launching an application, the Config of Executor's ActorSytem is composed of Worker's cluster config and application's config, and Worker's ```akka.scheduler.tick-duration``` could be overwrote by a wrong value. So another check here is needed.